src/components/ShippingDetails/ShippingDetails.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+
+//TODO: Decide about i18n solution or simply a JSON file where we can load translations.
+
+const TRANSLATIONS = {
+  SHIP_TO_ME: 'Ship to Me',
+  ARRIVES: 'Arrives'
+};
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+`;
+Container.displayName = 'Container';
+
+//TODO:PLACEHOLDER FOR ICON.
+const Icon = styled.div`
+  width: 50px;
+  background-color: lightblue;
+`;
+
+//TODO:PLACEHOLDER FOR SHIPPING TRUCK.
+const Arrow = styled.a`
+  width: 50px;
+  background-color: red;
+`;
+
+const MessageContainer = styled.div`
+  flex-grow: 1;
+  padding: 0 1rem;
+`;
+
+const ArrivalDates = styled.span`
+  font-weight: bold;
+`;
+
+const ShippingDetails = ({ minDate, maxDate, onClick, shippingPromo }) => (
```

Comment:
Not really sure what's the purpose of **onClick** here.

Reply:
Check the production website, when you click it it redirects to other page. So onClick has to be here

Reply:
We stated multiple times it is not a project purpose to mirror their site 100%. They are interested more in improving their process, so this website is just concrete, hands-on implementation.
On the other hand if you wanted to redirect to other page you can just use regular **<a>** tag.